### PR TITLE
COM-3527 - System didn't log shipping method change in the audit history entries

### PR DESCRIPTION
**Pull request by kibo-DeepakTajane**
updated the quote-history.hypr.live to include the missing field for ShippingMethod on the UI

### CODE REVIEW

File: templates/modules/b2b-account/quotes/quote-history.hypr.live [hunk 1]
```diff
@@ -90,6 +90,23 @@
...
+                                <td>
+                                  {% if change.type =='Add' %} {{labels.added}} {% elif change.type =='Remove' %} {{labels.removed}} {% else %} {{labels.updated}} {% endif %}
+                                  {{ changePath[0] }}
+                                </td>
+                              </tr>
+                              <tr>
+                                {% set shippingMethodName=change.fields|findwhere("name", "ShippingMethodName") %}
+                                <td>Shipping Methods</td>
+                                <td class="address">
+                                  {% include "modules/b2b-account/quotes/quote-history-field" with name=shippingMethodName value=shippingMethodName.oldValue %}
```
Comment (sagarraut0007) [inline templates/modules/b2b-account/quotes/quote-history.hypr.live]: this looks wrong to me, you are passing `name=shippingMethodName` which should be `name=shippingMethodName.name`. see how we have done this in other places.

File: templates/modules/b2b-account/quotes/quote-history.hypr.live [hunk 2]
```diff
@@ -90,6 +90,23 @@
...
+                                </td>
+                                <td class="space"></td>
+                                <td class="address">
+                                  {% include "modules/b2b-account/quotes/quote-history-field" with name=shippingMethodName value=shippingMethodName.newValue %}
```
Comment (sagarraut0007) [inline templates/modules/b2b-account/quotes/quote-history.hypr.live]: this looks wrong to me, you are passing `name=shippingMethodName` which should be `name=shippingMethodName.name`. see how we have done this in other places.

File: templates/modules/b2b-account/quotes/quote-history.hypr.live [hunk 3]
```diff
@@ -106,8 +123,7 @@
                                 {% set mobile=change.fields|findwhere("name", "Mobile") %}
 
                                 <td>
-                                  {% if change.type =='Add' %} {{labels.added}} {% elif change.type =='Remove' %} {{labels.removed}} {% else %} {{labels.updated}} {% endif %}
-                                  {{ changePath[0] }}
+                                  Address
```
Comment (sagarraut0007) [inline templates/modules/b2b-account/quotes/quote-history.hypr.live]: Add "Address"  into labels files (both en and de json). And FYI - if you add any new key in this file then add in alphabetical order. and then remove hardcoded label ("Address") and refer it from labels. see how we did in other places. ex . `{{labels.added}}`

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

File: templates/modules/b2b-account/quotes/quote-history.hypr.live (hunk 1 above)
Comment (kibo-DeepakTajane) [reply templates/modules/b2b-account/quotes/quote-history.hypr.live]: Done

File: templates/modules/b2b-account/quotes/quote-history.hypr.live (hunk 2 above)
Comment (kibo-DeepakTajane) [reply templates/modules/b2b-account/quotes/quote-history.hypr.live]: Done

File: templates/modules/b2b-account/quotes/quote-history.hypr.live (hunk 3 above)
Comment (kibo-DeepakTajane) [reply templates/modules/b2b-account/quotes/quote-history.hypr.live]: Done

File: labels/de-DE.json [hunk 4]
```diff
@@ -527,6 +527,8 @@
   "quoteExpiredBuyerBlob2": "Bitte senden Sie eine neue Anfrage oder wenden Sie sich an Ihren Vertriebsmitarbeiter, um Hilfe zu erhalten",
   "quoteExpiredSeller": "Bonjour, <br /> <br /> <br /> le devis \"{0}\" (# {1}) a expiré le",
   "quoteHistory": "Zitat Geschichte",
+  "quoteShippingMethods": "Versandarten",
```
Comment (sagarraut0007) [inline labels/de-DE.json]: Deepak, read my previous comment, where I mentioned these keys should be in alphabetical order. All the keys in this file are in alphabetical order. see "quoteExpiredBuyerBlob2" is first and then "quoteExpiredSeller".

File: labels/en-US.json [hunk 5]
```diff
@@ -630,6 +630,8 @@
   "quoteExpiredBuyerBlob2": "Please submit a new request or contact your sales representative for help",
   "quoteExpiredSeller": "Hi, <br /><br /><br /> Quote \"{0}\" (#{1}) has expired as of",
   "quoteHistory": "Quote History",
+  "quoteShippingMethods": "Shipping Methods",
```
Comment (sagarraut0007) [inline labels/en-US.json]: same keys should be in alphabetical order

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

File: labels/de-DE.json (hunk 4 above)
Comment (kibo-DeepakTajane) [reply labels/de-DE.json]: done

File: labels/en-US.json (hunk 5 above)
Comment (kibo-DeepakTajane) [reply labels/en-US.json]: done

**Review by sagarraut0007 — APPROVED** [review]

**Review by kibo-raghu — APPROVED** [review]
Approved

**Pull request merged**